<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>182</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="title">
       <string>Measurement</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="LoadSeq">
         <property name="text">
          <string>Load sequence</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="NewSeq">
         <property name="text">
          <string>New sequenece</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="AddSeqItem">
         <property name="text">
          <string>Secondary control</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ClearSeq">
         <property name="text">
          <string>Clear sequence</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="DataSave">
         <property name="text">
          <string>Data directory and name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="QueList">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="StartSeq">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(85, 255, 0);
font: 75 16pt &quot;Arial&quot;;</string>
         </property>
         <property name="text">
          <string>Run</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PauseSeq">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>16</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 0, 0);
background-color: rgb(255, 244, 246);</string>
         </property>
         <property name="text">
          <string>Pause</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="AbortSeq">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 0, 0);</string>
         </property>
         <property name="text">
          <string>Abort</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>182</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
